src/audio/channel_map.c

```diff
@@ -0,0 +1,33 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2019 Intel Corporation. All rights reserved.
+//
+// Author: Slawomir Blauciak <slawomir.blauciak@linux.intel.com>
+
+#include <ipc/channel_map.h>
+#include <sof/bit.h>
+#include <stdint.h>
+#include <stdlib.h>
+
+static uint32_t get_channel_map_size(struct sof_ipc_channel_map *chmap)
+{
+	return sizeof(*chmap) + (__builtin_popcount(chmap->ch_mask) *
```

Comment:
`__builtin_popcount` imho stuff like that should be in some math apis, but for now please add it to sof/common.h as other compiler-dependant builtins are there

Reply:
+1